bundles/org.openhab.binding.km200/src/main/java/org/openhab/binding/km200/internal/KM200Comm.java

```diff
@@ -126,12 +121,8 @@ public Integer sendDataToService(String service, byte[] data) {
 
                 // Create a method instance.
                 contentResponse = httpClient.newRequest("http://" + remoteDevice.getIP4Address() + service)
-                        .method(HttpMethod.POST)
-                        .agent("TeleHeater/2.2.3")
-                        .accept("application/json")
-                        .content(new BytesContentProvider(data))
-                        .timeout(5, TimeUnit.SECONDS)
-                        .send();
+                        .method(HttpMethod.POST).agent("TeleHeater/2.2.3").accept("application/json")
```

Comment:
😞

Reply:
What's the problem here?

Reply:
The eclipse formatter which we configured / decided upon 🤔